index.bs

```diff
@@ -1874,6 +1882,585 @@ A few abstract operations are used in this specification for utility purposes. W
   1. Return Record{[[size]]: size, [[highWaterMark]]: highWaterMark}.
 </pre>
 
+<h2 id="rbs">Readable Byte Streams</h2>
+
+<h3 id="rbs-intro">Using Readable Byte Streams</h3>
+
+<div class="example">
+  Unlike the readable stream, the readable byte stream provides multiple different kinds of readers. One is the
+  <code>ReadableByteStreamReader</code>. It has the same methods as the <code>ReadableStreamReader</code>, and can be
+  read in the same manner as the <code>ReadableStreamReader</code>. Other readers dedicated for the readable byte stream
+  are to be introduced into this spec.
+</div>
+
+<h3 id="rbs-class">Class <code>ReadableByteStream</code></h3>
+
+<h4 id="rbs-class-definition">Class Definition</h4>
+
+<em>This section is non-normative.</em>
+
+If one were to write the <code>ReadableByteStream</code> class in something close to the syntax of [[!ECMASCRIPT]], it
+would look like
+
+<pre><code class="lang-javascript">
+  class ReadableByteStream {
+    constructor(underlyingByteSource = {})
+
+    cancel(reason)
+    getReader()
+    pipeThrough({ writable, readable }, options)
+    pipeTo(dest, { preventClose, preventAbort, preventCancel } = {})
+    tee()
+  }
+</code></pre>
+
+<h4 id="rbs-internal-slots">Internal Slots</h4>
+
+Instances of <code>ReadableByteStream</code> are created with the internal slots described in the following table:
+
+<table>
+  <thead>
+    <tr>
+      <th>Internal Slot</th>
+      <th>Description (<em>non-normative</em>)</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>\[[closedPromise]]
```

Comment:
Hmm, why is [[closedPromise]] here instead of on the reader? Sorry if the answer becomes obvious after reading further.